### DISCUSSION TITLE
Does the "TX Power" Setting works with DM-1701 in the Channel Settings?

### DISCUSSION THREAD

**Discussion by skipper-henrik** [hmatuschek/qdmr#496] [Q&A] [answered]
To set my TX power correct, I have to use the Baofeng CPS in a VirtualBox. The DM-1701 does offer only Low Mid and Hight but qDMR shows me five levels which I can't see assign in any way...

**Comment by hmatuschek** [comment, marked as answer]
qdmr has more power options to match other devices also supported. The settings Min, Low, Med are mapped to DM-1701s "low" and qdmrs High & Max are mapped to DM1701s "high" settings. I do not own a DM1701, so I cannot test it. Let me know, if this works for you. Otherwise, I move this discussion to the issues tab.
